sky/catalog/runpod_catalog.py

```diff
@@ -12,7 +12,11 @@
 if typing.TYPE_CHECKING:
     from sky.clouds import cloud
 
-_df = common.read_catalog('runpod/vms.csv')
+# Runpod has no set updated schedule for their catalog. We pull the catalog
+# every 7 hours to make sure we have the latest information.
+_PULL_FREQUENCY_HOURS = 7
+_df = common.read_catalog('runpod/vms.csv',
```

Comment:
are you actually rerunning the script every 7 hours though or just pulling from the csv? It may be helpful to review if it's actually running the job again because otherwise it wouldn't be updating.

Reply:
Once this script is merged, we will add a new github workflow in our other repo https://github.com/skypilot-org/skypilot-catalog, and there we will run the script every 7h.